src/Sylius/Bundle/ApiBundle/Command/Payum/PayumRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Payum;
+
+use Sylius\Bundle\ApiBundle\Command\CommandAwareDataTransformerInterface;
+
+/** experimental */
+class PayumRequest implements CommandAwareDataTransformerInterface
+{
+    public function __construct(private string $payum_token)
```

Comment:
The format of this parameter have to be underscored because Payum is building URL with this format only